source/extensions/stat_sinks/wasm/BUILD

```diff
@@ -0,0 +1,35 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_extension",
+    "envoy_cc_library",
+    "envoy_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+# Stats sink for wasm
+
+envoy_package()
+
+envoy_cc_extension(
+    name = "config",
+    srcs = ["config.cc"],
+    hdrs = ["config.h"],
+    security_posture = "data_plane_agnostic",
+    deps = [
+        ":wasm_stat_sinks_impl",
```

Comment:
this code doesn't compile

```
$ bazel build //source/extensions/stat_sinks/wasm:config

ERROR: /envoy-wasm/source/extensions/stat_sinks/wasm/BUILD:14:1: in deps attribute of cc_library rule //source/extensions/stat_sinks/wasm:config: target '//source/extensions/stat_sinks/wasm:wasm_stat_sinks_impl' does not exist. Since this rule was created by the macro 'envoy_cc_extension', the error might have been caused by the macro implementation
```